lib/src/Uncertainty/Model/SklarCopula.cxx

```diff
@@ -253,8 +253,7 @@ Point SklarCopula::computeQuantile(const Scalar prob,
                                    const Bool tail) const
 {
   const UnsignedInteger dimension = getDimension();
-  const Scalar epsilon = cdfEpsilon_;
-  if (prob < -epsilon || prob > 1.0 + epsilon) throw InvalidArgumentException(HERE) << "Error: cannot compute a quantile for a probability level outside of [0, 1]";
+  if (!((prob >= 0.0) && (prob <= 1.0))) throw InvalidArgumentException(HERE) << "Error: cannot compute a quantile for a probability level outside of [0, 1]";
```

Comment:
!((prob >= -epsilon) && (prob <= 1.0+epsilon)
We have to keep a slight margin for numerical purpose here, because in this class most of the computations are done using integration and it has to be kept into account here.